test/src/test/kotlin/graphql/nadel/tests/next/fixtures/defer/MultipleFieldsinSingleDeferDirectiveTestSnapshot.kt

```diff
@@ -0,0 +1,104 @@
+import graphql.nadel.tests.next.TestSnapshot
+import graphql.nadel.tests.next.listOfJsonStrings
+import kotlin.Suppress
+import kotlin.collections.List
+import kotlin.collections.listOf
+
+/**
+ * This class is generated. Do NOT modify.
+ *
+ * Refer to [graphql.nadel.tests.next.UpdateTestSnapshots
+ */
+@Suppress("unused")
+public class MultipleFieldsinSingleDeferDirectiveTestSnapshot : TestSnapshot() {
+    override val calls: List<ExpectedServiceCall> = listOf(
+            ExpectedServiceCall(
+                service = "defer",
+                query = """
+                | {
+                |   defer {
+                |     fastField
+                |     ... @defer {
+                |       slowField
+                |       anotherSlowField
+                |     }
+                |   }
+                | }
+                """.trimMargin(),
+                variables = "{}",
+                response = """
+                | {
+                |   "defer": {
+                |     "fastField": "123"
+                |   }
+                | }
+                """.trimMargin(),
+                delayedResponses = listOfJsonStrings(
+                    """
+                    | {
+                    |   "hasNext": false,
+                    |   "incremental": [
+                    |     {
+                    |       "path": [
+                    |         "defer"
+                    |       ],
+                    |       "data": {
+                    |         "slowField": "slowString",
+                    |         "anotherSlowField": 123456789
+                    |       }
+                    |     }
+                    |   ]
+                    | }
+                    """.trimMargin(),
+                ),
+            ),
+        )
+
+    /**
+     * ```json
+     * {
+     *   "data": {
+     *     "defer": {
+     *       "slowField": "slowString",
+     *       "anotherSlowField": 123456789
+     *     }
+     *   }
+     * }
+     * ```
+     */
+    override val response: ExpectedNadelResponse = ExpectedNadelResponse(
+            response = """
+            | {
+            |   "data": {
+            |     "defer": {
+            |       "fastField": "123"
+            |     }
+            |   },
+            |   "hasNext": true
+            | }
+            """.trimMargin(),
+            delayedResponses = listOfJsonStrings(
+                """
+                | {
+                |   "hasNext": false,
+                |   "incremental": [
+                |     {
+                |       "path": [
+                |         "defer"
+                |       ],
+                |       "data": {
+                |         "slowField": "slowString",
+                |         "anotherSlowField": 123456789
+                |       }
+                |     }
+                |   ]
+                | }
+                """.trimMargin(),
+            ),
+        )
```

Comment:
Some more test cases you could add:
- defer on a field from a type returned as part of a list
- defer on calls to multiple underlying services 
- error cases - what if the initial response results on an error. What if one or more incremental response results in an error?
- extensions - on initial payload and incremental payloads